SubArray_sum_equals_k

```diff
@@ -0,0 +1,19 @@
+class Solution {
+    public int subarraySum(int[] nums, int k) {
+        int count =0;
```

Comment:
Please try the solution in class. You will find that there is a much better solution for this problem. Also please provide all the required information requested including the explanation of your approach